tasks/jsinspect.js

```diff
@@ -24,9 +25,21 @@ module.exports = function(grunt) {
       identifiers: false,
       failOnMatch: true,
       suppress:    100,
-      reporter:    'default'
+      reporter:    'default',
+      configFile:  '.jsinspectrc'
     });
 
+    if (fs.existsSync(options.configFile) && fs.lstatSync(options.configFile).isFile()) {
+      var contents = strip(fs.readFileSync(options.configFile, 'utf8'));
+      var rc = JSON.parse(contents);
```

Comment:
jsinspect had it but I took away, since thought that if is breaks, it should break..
Can add that back anyhow